integration_tests/integration/forbidden.spec.ts

```diff
@@ -0,0 +1,22 @@
+import Page from '../pages/page'
+import ForbiddenPage from '../pages/forbidden'
+
+context('Forbidden', () => {
+  beforeEach(() => {
+    cy.task('stubSetup')
+    cy.signIn()
+    cy.task('stubForLaoStatus403', { crn: 'J678910' })
+    cy.task('stubGetUnallocatedCase')
+    cy.visit('/pdu/PDU1/J678910/convictions/1/case-view', { failOnStatusCode: false })
+  })
+
+  it('403 message should be shown', () => {
+    const forbiddenPage = Page.verifyOnPage(ForbiddenPage)
+    forbiddenPage.message().should('exist')
+  })
+
+  it('403 header/title should be shown', () => {
```

Comment:
Do we really need two tests for this. Couldn't we check the message and the heading in the same test '403 page should be shown'

Reply:
fixed!

Reply:
Have you pushed the change?